include/sdbus-c++/Types.h

```diff
@@ -225,8 +224,11 @@ namespace sdbus {
 
         UnixFd& operator=(const UnixFd& other)
         {
+            if (this == &other) {
```

Comment:
Style convention appears to be:
```suggestion
            if (this == &other)
            {
```